R/modify.R

```diff
@@ -254,11 +254,39 @@ show_header_names <- function(x, include_example, quiet) {
   if (any(str_detect(names(df_print), "^modify_stat_"))) {
     df_print <- df_print |>
       dplyr::rename_with(
-        .fn = ~ str_remove(., pattern = "^modify_stat_")|> paste0("*"),
+        .fn = ~ str_remove(., pattern = "^modify_stat_") |> paste0("*"),
         .cols = starts_with("modify_stat_")
       )
   }
 
+  # create class map
+  class_mapping <- c(
+    "integer" = "int",
+    "numeric" = "dbl",
+    "character" = "chr",
+    "factor" = "fct",
+    "logical" = "lgl",
+    "Date" = "Date",
+    "POSIXct" = "POSIXct",
+    "POSIXlt" = "POSIXlt"
+  )
+
+  # Identify data type of dynamic values
+  df_print <- df_print |>
+    dplyr::mutate(
+      across(
+        if (ncol(df_print) > 2) 3:ncol(df_print) else everything(),
+        ~ ifelse(
+          !is.na(.),
+          paste0(
+            ., " <",
+            sapply(class(.), function(cls) class_mapping[cls]),
```

Comment:
I've updated the function so it evaluates the return of the `class(.)` section and maps the first "valid" class. So if it returned a vector, it would only paste 1 value in the <>